.changeset/heavy-meals-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/sdk': patch
```

Comment:
That's a breaking change, right @br41nslug?
```suggestion
'@directus/sdk': major
```

Reply:
i dont think its a breaking change, it is backward compatible ryt?

Reply:
Ah right, `keys` still works 👍 

We might make it a minor then, if we treat it as a new feature (@br41nslug 👀)
```suggestion
'@directus/sdk': minor
```

Reply:
I personally consider it to just another bug fix so `patch` seems fine to me. `minor` seems fine to as improvement to be better in sync with the API but a `major` it is surely not.

Reply:
Yeah, we can certainly consider it a patch too 👍